draft-ietf-ppm-dap.md

```diff
@@ -2411,6 +2430,37 @@ to the end as needed to produce the final aggregate output.
 Collectors must also maintain state for the lifetime of each task, which
 includes key material associated with the HPKE key configuration.
 
+## Aggregation Jobs & Replay Protection
+
+### Leader
+
+As part of the aggregation protocol, the Leader is responsible for generating
```

Comment:
nit: consistency. I'd be cool with saying "protocol" instead of "sub-protocol", but let's track in an issue.
```suggestion
As part of the aggregation sub-protocol, the Leader is responsible for generating
```

Reply:
👍🏻 It turned out there were a bunch of places that got this wrong -- I fixed them all up. There were a few places where it was ambiguous; for the most part I made as few changes as possible while still referring to "sub-protocols" when discussing only the upload/aggregation/collection portions of DAP.